themes/atomic/atomic.theme.bash

```diff
@@ -27,6 +27,8 @@ Face="\342\230\273"
 ## Parsers ##
 #############
 
+isFunction() { [[ "$(declare -Ff "$1")" ]]; }
```

Comment:
You shouldn't have to declare your own function for this. The `base` plugin already has a function called `command_exists` that does exactly that: https://github.com/Bash-it/bash-it/blob/master/plugins/available/base.plugin.bash#L154